python/lsst/display/matplotlib/matplotlib.py

```diff
@@ -436,10 +441,14 @@ def _dot(self, symb, c, r, size, ctype,
             x                Draw an x
             *                Draw a *
             o                Draw a circle
-            @:Mxx,Mxy,Myy    Draw an ellipse with moments (Mxx, Mxy, Myy) (argument size is ignored)
-            An afwGeom.ellipses.Axes Draw the ellipse (argument size is ignored)
-    Any other value is interpreted as a string to be drawn. Strings obey the fontFamily (which may be extended
-    with other characteristics, e.g. "times bold italic".  Text will be drawn rotated by textAngle
+            @:Mxx,Mxy,Myy    Draw an ellipse with moments (Mxx, Mxy, Myy)
+                             (argument size is ignored)
+            An afwGeom.ellipses.Axes Draw the ellipse (argument size is
```

Comment:
"is ignored)" belongs with "Draw the ellipse".  It may make sense to expand the left column.